src/server/llmQueries.ts

```diff
@@ -49,145 +61,183 @@ type LanguageCode = "nn" | "nb" | "en";
 type ArticleSummaryQuery = Record<LanguageCode, (text: string, title: string) => string>;
 
 const ARTICLE_SUMMARY_QUERY: ArticleSummaryQuery = {
-  nb: (text: string, title: string) => `<draft> ${text} </draft>
+  nb: (text: string, title: string) => `
+        <draft> ${text} </draft>
         <Instructions>
           Du har lang erfaring fra utdanningssektoren.
           Du har spesialisert deg i å oppsummere fagartikler for å gjøre dem mer tilgjengelige for et bredere publikum.
           Du har fått som oppdrag å hjelpe med å forbedre læringsopplevelsen for elevene.
           For å gjøre dette må du lese gjennom og lage en oppsummering av artikler sånn at videregåendestudentene lett kan få et inntrykk av de viktigste poengene i artikkelen.
-          Du har fått som oppdrag å lese gjennom artikkelen i <draft> med tittel "${title}" og skrive en oppsummering på NB. Oppsummeringen skal være på mindre enn 300 ord.
-          Svaret skal leveres med refleksjoner på hvorfor du har valgt å inkludere de punktene du har valgt først, så selve oppsummeringen skrevet i en <answer>-tag.
-          Om du ikke klarer å levere svaret på formatet <answer> som er spesifisert skal første linje i responsen være <ERROR>-tag.
+          Du har fått som oppdrag å lese gjennom og skrive en oppsummering på NB til artikkelen i <draft> tagget, tittelen til artikkelen er "${title}". 
+          Oppsummeringen skal være på mindre enn 300 ord og skrives i en <answer>-tag.
+          ${DEBUG_INSTRUCTION.nb}
+          ${ERROR_INSTRUCTION.nb}
         </Instructions>`,
-  en: (text: string, title: string) => `<draft>${text}</draft>
+  en: (text: string, title: string) => `
+        <draft>${text}</draft>
         <Instructions>
           You have an extensive experience from the educational sector.
           You are specialized in summarizing educational articles to make them more accessible for a wider audience.
           Your task is to help improving the learning experience for the students.
           To achieve this you have to read and summarize articles in a way that high school students can easily gain an impression of the most important points in the article.
-          Your task is to read through the article in <draft> with title "${title}" and write a summary in EN. The summary must be in less than 300 words.
-          The response must be given with reflections as to why you have chosen to include the selected points first, then the summary in an <answer>-tag.
-          If you cannot deliver the answer in the given format <answer>, the first line in the response should be <ERROR>-tag.
+          Your task is to read through and write a summary in EN to the article within the <draft> tag, the title of the article is "${title}". 
+          The summary must be in less than 300 words and written inside a <answer>-tag.
```

Comment:
`The summary must be shorter than 300 words, and written inside an <answer> tag`